test/slider-test.js

```diff
@@ -167,9 +156,9 @@ describe('Slider API', () => {
       const slider = newSlider();
       const state = slider.getPublicState();
 
-      assert.isTrue(state.hasOwnProperty('max'), 'max exists');
-      assert.isTrue(state.hasOwnProperty('min'), 'min exists');
-      assert.isTrue(state.hasOwnProperty('values'), 'values exists');
+      assert.isTrue({}.hasOwnProperty.call(state, 'max'), 'max exists');
```

Comment:
@lencioni before you dig too far into this PR, a lot of this sort of thing has been resolved in the other PR and the work @ljharb did last night. Probably worth waiting, and I can ping you once this has been rebased.

I will, however, implement the changes you suggested above. I just don't want to waste your cycles.